finality-aleph/src/network.rs

```diff
@@ -603,7 +595,7 @@ where
 
     pub async fn run(mut self) {
         let mut network_event_stream = self.network.event_stream();
-        let mut status_ticker = tokio::time::interval(Duration::from_secs(10));
+        let mut status_ticker = IntervalStream::new(tokio::time::interval(Duration::from_secs(10)));
```

Comment:
Is this necessary? Previous version with `tick` instead of `next` wouldn't work?

Reply:
you're right is equivalent and we can get rid of one dependency - changed